src/app/shell/personal-cabinet/provider/create-provider-admin/create-provider-admin.component.ts

```diff
@@ -110,10 +138,14 @@ export class CreateProviderAdminComponent extends CreateFormComponent implements
             isActive: false,
             disable: false,
           },
-          {
-            name: this.providerRole == providerAdminRole.deputy ?
-              NavBarName.CreateProviderDeputy : 
-              NavBarName.CreateProviderAdmin,
+          { 
```

Comment:
Please try to avoid nested ternary operators ( it's not readable to everyone )
I would suggest to probably stick to if/else or switch when working on a team.